packages/devextreme/js/__internal/ui/chat/messagebox.ts

```diff
@@ -81,10 +99,13 @@ class MessageBox extends DOMComponent<MessageBox, Properties> {
       autoResizeEnabled: true,
       valueChangeEvent: 'input',
       maxHeight: '8em',
-      onInput: (): void => {
+      onInput: (e: InputEvent): void => {
         const shouldButtonBeDisabled = !this._isValuableTextEntered();
 
         this._toggleButtonDisableState(shouldButtonBeDisabled);
+
+        this._triggerTypingStartAction(e);
```

Comment:
Just clarify: Did you consider scenanario with firing the typing end event immediately when the input is empty?

Reply:
We don't have that scenario. Scenarios that exist:

1. the user enters or deletes something in the input field
2. the user sends a message